modules/home-manager/sops.nix

```diff
@@ -174,6 +171,18 @@ in {
       description = "What to log";
     };
 
+    environment = lib.mkOption {
+      type = lib.types.attrsOf (lib.types.either lib.types.str lib.types.path);
+      default = {};
+      description = ''
+        Environment variables to set before calling sops-install-secrets.
+
+        The values are placed in single quotes and not escaped any further to
+        allow usage of command substitutions for more flexibility. To properly quote
```

Comment:
Is this actually true? I don't think systemd will do command substitution.

Reply:
I don't now about launchd, but it also looks to me that it should be the same.

Reply:
That may not be true. I copied the description for the environment option from the NixOS module at modules/sops/default.nix.

I also have only tested the launchd code path. It does work on macOS. I do not have access to a NixOS linux system. Hopefully someone can test the systemd code path.